Update local development guide (#13023)
supabase/supabase: apps/docs/pages/guides/cli/local-development.mdx

supabase start
```

Once all of the Supabase services are running, you'll see output containing your local Supabase credentials. It should look like this, with urls and keys that you'll use in your local project:

```

Started supabase local development setup.

         API URL: http://localhost:54321
          DB URL: postgresql://postgres:postgres@localhost:54322/postgres
      Studio URL: http://localhost:54323
    Inbucket URL: http://localhost:54324
        anon key: eyJh......
service_role key: eyJh......

```

You can use the [supabase stop](/docs/reference/cli/usage#supabase-stop) command at any time to stop all services.

## Access your project's services

> You can now visit your local Dashboard at [http://localhost:54323](http://localhost:54323), and access the services directly with any Postgres client or through the API Gateway ([Kong](https://github.com/Kong/kong)).

Author: Humm yeah actually not sure why I left that in from the original. I have no idea what it means. 

Think we should expand or remove?

> You can now visit your local Dashboard at [http://localhost:54323](http://localhost:54323), and access the services directly with any Postgres client or through the API Gateway ([Kong](https://github.com/Kong/kong)).

Reviewer: ```suggestion
You can now visit your local Dashboard at [http://localhost:54323](http://localhost:54323), and access the database directly with any Postgres client via `postgresql://postgres:postgres@localhost:54322/postgres`.
```

  ></iframe>
</div>

For this guide, we'll create a table called `employees` and see how we can make changes to it.

<StepHikeCompact>

  <StepHikeCompact.Step step={1}>
    <StepHikeCompact.Details title="Create your first migration file">

      To get started, generate a [new migration](https://supabase.com/docs/reference/cli/supabase-migration-new) to store the SQL needed to create our `employees` table

    </StepHikeCompact.Details>

    <StepHikeCompact.Code>

```bash
supabase migration new create_employees_table
```

    </StepHikeCompact.Code>

  </StepHikeCompact.Step>
</StepHikeCompact>

<StepHikeCompact>

  <StepHikeCompact.Step step={2}>
    <StepHikeCompact.Details title="Add the SQL to your migration file">
      This creates a new migration: supabase/migrations/\<timestamp\>
        _create_employees_table.sql.

        To that file, add the SQL to create this `employees` table
    </StepHikeCompact.Details>

    <StepHikeCompact.Code>
      ```sql

> create table

Author: I think we should think about this sql statement as its own "how-to" or reference. 

A user should ideally be able to create a new table from this example really easily. What other data types should we include in here that would be useful for people? 

Should the id primary key be an integer or something else? A uuid?

Author: suggestions here: 
an auto-incrementing id (serial?) 
a number
a date

      This creates a new migration file: supabase/migrations/\<timestamp\>
        _add_department_to_employees_table.sql.

        To that file, add the SQL to create a new department column
    </StepHikeCompact.Details>

    <StepHikeCompact.Code>
      ```sql
alter table
  if exists public.employees add department text default 'Hooli';
````

    </StepHikeCompact.Code>

  </StepHikeCompact.Step>
</StepHikeCompact>

### Add sample data

Now that you are managing your database with migrations scripts, it would be great have some seed data to use every time you reset the database.

For this, you can use the seed script in `supabase/seed.sql`. This file was automatically created when you ran [`supabase init`](/docs/reference/cli/usage#supabase-init)) at the beginning.

<StepHikeCompact>

  <StepHikeCompact.Step step={1}>
    <StepHikeCompact.Details title="Populate your table">
      Insert data into your `employees` table with your `supabase/seed.sql` file.
    </StepHikeCompact.Details>

    <StepHikeCompact.Code>
      ```sql

-- in supabase/seed.sql
insert into
public.employees (name)
values
('Erlich Bachman'),
('Richard Hendricks'),

> ('Monica Hall');

Author: I see a mix of bigint and serial for ids. Any preference there?

> ('Monica Hall');

Author: wrong comment there ^^ 

good call on email being not null, I'll adjust

> ('Monica Hall');

Reviewer: For `id` column I think we want:
```sql
id bigint primary key generated always as identity
```
Sounds like `serial` and `bigserial` are no longer recommended.

  ></iframe>
</div>

For this guide, we'll create a table called `employees` and see how we can make changes to it.

<StepHikeCompact>

  <StepHikeCompact.Step step={1}>
    <StepHikeCompact.Details title="Create your first migration file">

      To get started, generate a [new migration](https://supabase.com/docs/reference/cli/supabase-migration-new) to store the SQL needed to create our `employees` table

    </StepHikeCompact.Details>

    <StepHikeCompact.Code>

```bash
supabase migration new create_employees_table
```

    </StepHikeCompact.Code>

  </StepHikeCompact.Step>
</StepHikeCompact>

<StepHikeCompact>

  <StepHikeCompact.Step step={2}>
    <StepHikeCompact.Details title="Add the SQL to your migration file">
      This creates a new migration: supabase/migrations/\<timestamp\>
        _create_employees_table.sql.

        To that file, add the SQL to create this `employees` table
    </StepHikeCompact.Details>

    <StepHikeCompact.Code>
      ```sql

create table
employees (
id bigint primary key generated always as identity,
created_at timestamp with time zone default now(),
name text,
email text,
created_at timestampz default now()

> );

Reviewer: Heads up, we got `created_at` twice now.

Author: oof, I decide to try vscode's new merge editor one single time and get this!
